repos/spack_repo/builtin/packages/relion/package.py

```diff
@@ -110,15 +115,16 @@ class Relion(CMakePackage, CudaPackage):
 
     def cmake_args(self):
         args = [
-            "-DCMAKE_C_FLAGS=-g",
-            "-DCMAKE_CXX_FLAGS=-g",
             "-DGUI=%s" % ("+gui" in self.spec),
             "-DDoublePrec_CPU=%s" % ("+double" in self.spec),
             "-DDoublePrec_GPU=%s" % ("+double-gpu" in self.spec),
             "-DALLOW_CTF_IN_SAGD=%s" % ("+allow_ctf_in_sagd" in self.spec),
             "-DMKLFFT=%s" % ("+mklfft" in self.spec),
             "-DALTCPU=%s" % ("+altcpu" in self.spec),
         ]
+        if spec.satisfies("+gui"):
```

Comment:
@snehring, 

while refactoring to make the commits applicable without conflicts to both base commits of the PRs, I forgot to add small adaptation for my originally refactored code, so `spec` needs to be changed to `self.spec` here, because in the new refactor, I did not add `spec = self.spec`, which made this originally work (and allowed to shorten `self.spec` to `spec`, but that's no longer included here:)

```suggestion
        if self.spec.satisfies("+gui"):
```